src/Hangfire.Core/Dashboard/Pager.cs

```diff
@@ -103,13 +118,11 @@ private ICollection<Item> GenerateItems()
 
             return pagerItems;
         }
-
```

Comment:
Please have a new line after every closing brace. Think as it like a paragraph you always have a new line when you end one.